interfaces/builtin/framebuffer.go

```diff
@@ -83,17 +82,6 @@ func (iface *FramebufferInterface) AppArmorConnectedPlug(spec *apparmor.Specific
 
 // Getter for the security snippet specific to the plug
 func (iface *FramebufferInterface) ConnectedPlugSnippet(plug *interfaces.Plug, slot *interfaces.Slot, securitySystem interfaces.SecuritySystem) ([]byte, error) {
-	switch securitySystem {
```

Comment:
We have a work item to convert all interfaces to use proper udev tagging for our next sprint which will then land in a future snapd version 2.24/2.25/2.26 depending on complexity and timing.